### PR TITLE
Add an initial list of new features in Docker Engine 1.5.0

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@icecrime @fredlf @jamtur01

**Comment by jamtur01** [comment]
LGTM

**Comment by thaJeztah** [comment]
`docker build -f some/Dockerfile` might be something to mention, I know this was a much requested feature (don't have the PR number at hand)

File: docs/sources/release-notes.md [hunk 1]
```diff
@@ -2,78 +2,34 @@ page_title: Docker 1.x Series Release Notes
...
-*Client fixes*
- 
-* Fixed a regression related to STDIN redirection.
-* Fixed a regression involving `docker cp` when the current directory is the
-destination.
+* A new `docker stats CONTAINERID` command has been added to allow users to view a
+  continuously updating stream of container resource usage statistics. See the
+  [`stats` command line reference](/reference/commandline/cli/#stats) and the 
+  [container `stats` API reference](/reference/api/docker_remote_api_v1.17/#get-container-stats-based-on-resource-usage).
+* The Docker daemon has added support for IPv6 networking between containers
```
Comment (fredlf) [inline docs/sources/release-notes.md]: has added support for-->now supports

File: docs/sources/release-notes.md [hunk 2]
```diff
@@ -89,4 +45,4 @@ impact on users. This list will be updated as issues are resolved.
 An idiosyncrasy in AUFS prevents permissions from propagating predictably
 between upper and lower layers. This can cause issues with accessing private
 keys, database instances, etc. For complete information and workarounds see
-[Github Issue 783](https://github.com/docker/docker/issues/783).
\ No newline at end of file
+[Github Issue 783](https://github.com/docker/docker/issues/783).
```
Comment (fredlf) [inline docs/sources/release-notes.md]: Let's also add the Safari/Hub issue while we're in here:

** Docker Hub incompatible with Safari 8**

The Docker Hub has multiple issues running on Safari 8, the default browser for OS X 10.10 (Yosemite). Users should access the hub using a different browser.

**Comment by fredlf** [comment]
LGTM, with one sneaky addition and a copy edit.

File: docs/sources/release-notes.md (hunk 2 above)
Comment (thaJeztah) [inline docs/sources/release-notes.md]: Is that the styling issue? I think I tried some CSS changes a while ago that fixed it, but need to look if I made notes. Or are there other issues?

**Comment by SvenDowideit** [comment]
@crosbymichael @icecrime

File: docs/sources/release-notes.md (hunk 2 above)
Comment (SvenDowideit) [inline docs/sources/release-notes.md]: much otherwise. and yes, I too need to spend some time with the css - maybe next week

**Comment by fredlf** [comment]
LGTM

**Pull request merged**